Common/Interfaces/IAlgorithm.cs

```diff
@@ -133,6 +133,11 @@ AlgorithmStatus Status
             set;
         }
 
+        Dictionary<SecurityType, List<TickType>> AvailableDataFeeds
```

Comment:
Seems strange that the algorithm exposes available dates  feeds, but rather should probably be told what feeds are available.